www-src/app/replicator/conflicts_handler.coffee

```diff
@@ -11,7 +11,9 @@ module.exports = class ConflictsHandler
     handleConflicts: (doc, callback) ->
         # Get the doc with conflicts (and revs) infos from Pouch
         @db.get doc._id, { conflicts: true, revs: true }, (err, local) =>
-            return callback err if err
+            if err
+                return callback null, doc if err.status is 404
```

Comment:
If we get a 404 here, it means that we make this work too soon (ie before PouchDB store the changes)